include/mgmt_conn.h

```diff
@@ -105,6 +105,12 @@ void zinfo_destroy_cb(zvol_info_t *zinfo);
 void uzfs_zvol_mgmt_thread(void *arg);
 int finish_async_tasks(void);
 
+int uzfs_zvol_create_snapshot_update_zap(zvol_info_t *zinfo,
```

Comment:
variable name changes here in prototypes as well

Reply:
done